### DISCUSSION TITLE
How do I override a mergeable config?

### DISCUSSION THREAD

**Discussion by cajubelt** [renovatebot/renovate#25112] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I want to be able to extend config presets, but override `postUpdateOptions` to an empty list. 

### What you tried so far.

Here is my renovate config:
```
{
  extends: ['github>valora-inc/renovate-config:default.json5'],
  postUpdateOptions: [], // overriding because 'yarnDedupeFewer' causes breaking changes in this repo
}
```
The config I'm trying to extend is [here](https://github.com/valora-inc/renovate-config/blob/main/default.json5). It sets `postUpdateOptions` to `['yarnDedupeFewer']`. My understanding is that since `postUpdateOptions` is mergeable, my attempt to override it does not work, it just gets added to the preset value. 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Unfortunately there is not a way.

**Comment by msugakov** [comment]
Hi,

I have a similar question. Not sure if I should open a new issue for it.

Is `schedule` mergeable? The [docs](https://docs.renovatebot.com/configuration-options/#schedule) don't mention that.

Is there a way to see the eventual config after resolving all `extends` and merging/overriding all attributes?

Thanks!

**Reply by RahulGautamSingh** [reply]
1. Schedule is non-mergeable. 

2. Try [printConfig](https://docs.renovatebot.com/configuration-options/#printconfig)
